### ISSUE TITLE
The OMF_PATH should be quoted

### ISSUE THREAD

**Issue by dlancea** [oh-my-fish/oh-my-fish#197]
My install failed because my home directory contains a space in it. I had to manually edit the generated config.fish and add quotes around the generated OMF_PATH.

I believe this is an issue in this file:
https://github.com/oh-my-fish/oh-my-fish/blob/master/templates/config.fish

I believe it should read:
```
# Path to Oh My Fish install.
set -gx OMF_PATH "{{OMF_PATH}}"

# Customize Oh My Fish configuration path.
#set -gx OMF_CONFIG "{{OMF_CONFIG}}"
```

The OMF_PATH (and OMF_CONFIG) might contain a space or other special character, so these paths should be quoted to prevent install error.

**Comment by derekstavis** [comment]
Thanks for pointing :+1:

**Comment by oranja** [comment]
Closed with #219

**Comment by derekstavis** [comment]
Thanks @oranja!
